bin/syncProductFlags.ts

```diff
@@ -0,0 +1,28 @@
+import '../src/config';
+import createOrGetConnection from '../src/db';
+
+(async (): Promise<void> => {
+  const createdAfterArgument = process.argv[2];
```

Comment:
Idea is to run this script in batches to avoid deadlocks on the DB because we will have to update all the rows.